### DISCUSSION TITLE
Copied project

### DISCUSSION THREAD

**Discussion by ketanop321** [MAVRICK-1/e-commerce_website#480] [General]
This project copied from a youtube channel [Advanced UI Techniques](https://www.youtube.com/@rinkuv37)

**Comment by MAVRICK-1** [comment]
@Ketanop321 We have integrated many features, including OpenStreetMap, a redesigned API with location services, a seller form, a "My Account" page, and more. The initial base for the project was taken from the following source, with permission from the owner: [Nest Multivendor Organic Grocery - Laravel Ecommerce on CodeCanyon](https://codecanyon.net/item/nest-multivendor-organic-grocery-laravel-ecommerce/35109578). Additionally, we got a different backend to store data.

**Comment by MAVRICK-1** [comment]
@Ketanop321 one more thing this project was my teams hackthon project 
https://github.com/SHANKARA-CODERS/NestONDC. Here's the link of the GitHub repository.
It was created 5 months ago  . Hope you got your answer .
